subgraph.yaml

```diff
@@ -148,7 +78,7 @@ dataSources:
           handler: handleShelfRepay
       file: ./src/mapping.ts
   - kind: ethereum/contract
-    name: Threshold2
+    name: ThresholdStaging
```

Comment:
Instead of `Staging`, I think we should add semver versions or git tags. That way, it's much easier to manage multiple contracts and ABIs.